src/Parser/Mage/Fire/Modules/Features/CooldownTracker.js

```diff
@@ -0,0 +1,71 @@
+import CoreCooldownTracker, { BUILT_IN_SUMMARY_TYPES } from 'Parser/Core/Modules/CooldownTracker';
+
+import SPELLS from 'common/SPELLS';
+
+const debug = false;
+
+class CooldownTracker extends CoreCooldownTracker {
+  static cooldownSpells = [
+    ...CooldownTracker.cooldownSpells,
+    {
+      spell: SPELLS.COMBUSTION,
+      summary: [
+        BUILT_IN_SUMMARY_TYPES.DAMAGE,
+      ],
+    },
+    {
+      spell: SPELLS.RUNE_OF_POWER_TALENT,
+      duration: 10,
```

Comment:
I think I've discussed this with Sharrq. You basically have 2 options here. 

1) either leave Rune of Power in `cooldownSpells` in which case the `duration` is unnecessary. This will create a "cooldown window" every time you stand in Rune of Power. Therefore, if you leave and re-enter the Rune (removebuff and then applybuff again), it will create 2 "windows". 
2) if instead you put it in `castCooldowns`, you need to specify the `duration` and then it will create a fixed 10 second "window" on Rune of Power cast. The downside of this approach is that it tracks events that happen *outside* of the Rune of Power as well. 

So the choice is up to you what you want - `cooldownSpells` will be more accurate but potentially more messy, `castCooldowns` will be tidier but potentially less accurate.

Reply:
I'll be sticking with the cooldownSpells, it seems to be more accurate as if you are out of the rune we shouldn't be applying the buff to it.